test/browser_reporting.js

```diff
@@ -76,6 +76,12 @@ function reportTopLevelError(e) {
 }
 
 if (typeof window === 'object' && window) {
+  const urlString = window.location.search;
+  const searchParams = new URLSearchParams(urlString);
+  if (searchParams.has('capture_stdio')) {
```

Comment:
If the url has `capture_stdio=false` will this do the right thing? Not familiar with this API.

Reply:
Nah.... that would not do as one might expect, but I'm not sure I care enough to polish this feature that much.  Is it worth it?

Reply:
probably not, unless there exists some convenient better API like `searchParams.has_and_value_is_true()`

Reply:
Perhaps I'll drop the `=true` part of the url to avoid confusion.